ra_aid/llm.py

```
@@ -1,93 +1,184 @@
 from langchain_openai import ChatOpenAI
 from langchain_anthropic import ChatAnthropic
 from langchain_core.language_models import BaseChatModel
+from langchain_google_genai import ChatGoogleGenerativeAI
+from ra_aid.chat_models.deepseek_chat import ChatDeepseekReasoner
 
-def initialize_llm(provider: str, model_name: str, temperature: float | None = None) -> BaseChatModel:
-    """Initialize a language model client based on the specified provider and model.
 
-    Note: Environment variables must be validated before calling this function.
-    Use validate_environment() to ensure all required variables are set.
+def get_env_var(name: str, expert: bool = False) -> Optional[str]:
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)